src/components/EditorTopBar.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { AppServicesProvider } from "./AppServicesContext";
+import { EditorTopBar } from "./EditorTopBar";
+
+const appConfiguration = {
+  dopplerExternalUrls: {
+    home: "homeUrl",
+    campaigns: "campaignsUrl",
+    lists: "listsUrl",
+    controlPanel: "controlPanelUrl",
+  }
+};
+
+describe(EditorTopBar.name, () => {
+  it("should render Exit Button", async () => {
+    // Arrange
+    const appServices = { appConfiguration } as any;
+
+    // Act
+    render(
+      <AppServicesProvider
+      appServices={appServices}
+    >
+      <EditorTopBar onSave={() => {}}></EditorTopBar>
+      </AppServicesProvider>
+    );
+
+    // Assert
+    screen.getByText('Salir del Editor');
+  });
+
+  it("should render Exit Options after click on Exit Button", async () => {
+    // Arrange
+    const appServices = { appConfiguration } as any;
+
+    // Act
+    render(
+      <AppServicesProvider
+      appServices={appServices}
+    >
+      <EditorTopBar onSave={() => {}}></EditorTopBar>
+      </AppServicesProvider>
+    );
+
+    userEvent.click(screen.getByText('Salir del Editor'));
+
+    // Assert
+    expect(screen.queryByText('Inicio')).not.toBeNull;
+  });
+
+  it("should Exit to specific url", async () => {
+    // Arrange
+    const appServices = { appConfiguration } as any;
+
+    // Act
+    render(
+      <AppServicesProvider
+      appServices={appServices}
+    >
+      <EditorTopBar onSave={() => {}}></EditorTopBar>
+      </AppServicesProvider>
+    );
+
+    userEvent.click(screen.getByText('Salir del Editor'));
+
+    // Assert
+    expect(screen.getByText('Inicio').closest('a')).toHaveAttribute('href',appConfiguration.dopplerExternalUrls.home)
```

Comment:
What do you think about test all the options?

```suggestion
  it("should has the right URLs", async () => {
    // Arrange
    const appServices = { appConfiguration } as any;

    // Act
    render(
      <AppServicesProvider
      appServices={appServices}
    >
      <EditorTopBar onSave={() => {}}></EditorTopBar>
      </AppServicesProvider>
    );

    userEvent.click(screen.getByText('Salir del Editor'));

    // Assert
    const homeOption = screen.getByText('Inicio');
    const campaignsOption = screen.getByText('Campañas');
    const listsOption = screen.getByText('Listas');
    const controlPanelOption = screen.getByText('Panel de Control');

    expect(homeOption.closest('a'))
      .toHaveAttribute('href', appConfiguration.dopplerExternalUrls.home);

    expect(campaignsOption.closest('a'))
      .toHaveAttribute('href', appConfiguration.dopplerExternalUrls.campaigns);

    expect(listsOption.closest('a'))
      .toHaveAttribute('href', appConfiguration.dopplerExternalUrls.lists);

    expect(controlPanelOption.closest('a'))
      .toHaveAttribute('href', appConfiguration.dopplerExternalUrls.controlPanel);
```

Reply:
Thanks for your suggestion! Applied.